cmd/parquettest/parquettest.go

```diff
@@ -0,0 +1,140 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	parquet "github.com/parquet-go/parquet-go"
+)
+
+// TimeSeries represents a single time series data point with labels
+type TimeSeries struct {
+	Timestamp int64             `parquet:",delta"`
+	Value     float64           `parquet:",zstd"`
+	Labels    map[string]string `parquet:"labels"`
```

Comment:
is this going to map to columns? or a single column? I should probably read up on how map is represented in parquet